server/controller/expertise.js

```diff
@@ -71,11 +89,13 @@ exports.updateExpertise = async (req, res) => {
   }
 
   try {
-    const user = userDB.findOne({ authO_ref: nickname });
-    const expertise = expertiseDB.findOne({ topic, user_id: user._id });
+    let expertise = await expertiseDB.findOne({
+      _id: expertise_id,
```

Comment:
convert to objectid

Reply:
Mongoose converts it automatically if it's a valid ObjectID